test/e2e/authorizer/webhook-httest-pki/ca.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Should this have been committed? 🤔

Reply:
I have ran into some flaky tests, these are generated by `httest` tool.
Let me try again with generating them during the run.